apps/predbat/predbat.py

```diff
@@ -2146,7 +2146,7 @@ def download_predbat_releases_url(self, url):
             pdata = self.github_url_cache[url]["data"]
             age = now - stamp
             if age.seconds < (120 * 60):
-                self.log("Using cached GITHub data for {} age {} minutes".format(url, age.seconds / 60))
+                self.log("Using cached GITHub data for {} age {} minutes".format(url, round(age.seconds / 60,1)))
```

Comment:
I think this is the same as self.dp1(x) - we should probably pick one of them :)